### PR TITLE
Session by event.

### PR DESCRIPTION
**Pull request by clacote**
New enum ConferenceEvent (mixit12, mixit13), with const CURRENT giving current event.
All sessions are related to an event. A speaker is a member having a validated talk on current event only.
Sponsor are also related to one or several events.
Reset attendee registration (no tickets on current event). Previous attendee are given the new badge PreviousAttendee.

No GUI rework. Some screens, now empty, may need some editorial content (list of current sessions is empty for example). But as this PR is becoming really big, I push it now for review.

We can merge it, but we won't deploy it now.

### CODE REVIEW

File: app/models/Sponsor.java [hunk 1]
```diff
@@ -27,15 +31,35 @@
     @Enumerated(EnumType.STRING)
     public Level level;
 
-    public Sponsor(String login) {
+	@ElementCollection @Enumerated(EnumType.STRING)
```
Comment (cexbrayat) [inline app/models/Sponsor.java:34]: oh le code non formatté! :-)

Comment (clacote) [inline app/models/Sponsor.java:34]: La faute à Idea 12! :p

**Comment by cexbrayat** [comment]
Nice!

**Comment by clacote** [comment]
On est d'accord, il ne faut pas la déployer tout de suite : je n'ai pas
testé le script SQL de migration. Et y'a encore des trucs à fignoler
(genre que retourne la recherche?).

   Cyril.

**Pull request merged**